### ISSUE TITLE
[stable/dask] Image version bump to 1.1.5 esp for pandas version

### ISSUE THREAD

**Issue by javabrett** [helm/charts#13036]
**Version of Helm and Kubernetes**:
```
 kubectl version
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.7", GitCommit:"6f482974b76db3f1e0f5d24605a9d1d38fad9a2b", GitTreeState:"clean", BuildDate:"2019-03-27T15:15:05Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"12+", GitVersion:"v1.12.6-eks-d69f1b", GitCommit:"d69f1bf3669bf00b7f4a758e978e0e7a1e3a68f7", GitTreeState:"clean", BuildDate:"2019-02-28T20:26:10Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
 helm version
Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
```

**Which chart**:
`stable/dask`

**What happened**:
Default `values.yaml` uses `daskdev/dask:1.1.0` in 3 places:

https://github.com/helm/charts/blob/1d03bbee123a4e2baffed6d5af7237b949a07475/stable/dask/values.yaml#L6-L9

This image has `pandas==0.23.4`:

```
 docker run -it --rm daskdev/dask:1.1.0 bash -lc "pip freeze | grep pandas"
pandas==0.23.4
```

Tag 1.1.5 has:

```
 docker run -it --rm daskdev/dask:1.1.5 bash -lc "pip freeze | grep pandas"
pandas==0.24.2
```
... which is the latest `pandas`.

These versions are incompatible when pickling.  If a Dask `Client` is running on a machine with `pandas==0.24.2`, a Dask Helm Chart worker with default values and image tag will see this:

```
File "/opt/conda/lib/python3.7/site-packages/distributed/protocol/pickle.py", line 59, in loads return pickle.loads(x) ModuleNotFoundError: No module named 'pandas.core.internals.managers'; 'pandas.core.internals' is not a package
```

**What you expected to happen**:
The Dask Helm chart with default values should work with a `Client` running in an env with current Pandas.

Workaround: just edit `values.yaml` and move-up to `1.1.5`, but I figured an issue was warranted to ensure the version moves-up promptly.

**How to reproduce it** (as minimally and precisely as possible):
Send any Pandas dataframe to a worker deployed with the default Helm chart/values.

/cc @mrocklin .

**Comment by javabrett** [comment]
Should this change be attempted again with Dask `1.2.0`?  Not sure what the update-frequency policy is on stable charts.

**Comment by javabrett** [comment]
This will be fixed when #12552 is merged.
